Sources/ComposableArchitecture/SwiftUI/ActionSheet.swift

```diff
@@ -107,12 +107,12 @@ import SwiftUI
 public struct ActionSheetState<Action> {
   public let id = UUID()
   public var buttons: [Button]
-  public var message: String?
-  public var title: String
+  public var message: LocalizedStringKey?
+  public var title: LocalizedStringKey
 
   public init(
-    title: String,
-    message: String? = nil,
+    title: LocalizedStringKey,
+    message: LocalizedStringKey? = nil,
```

Comment:
Dunno if we want `S: StringProtocol` overloads for all these initializers, too...